format/midi/testdata/Makefile

```diff
@@ -16,8 +16,10 @@ run: build
 	go run . -d midi dv format/midi/testdata/reference.mid
 
 debug: build
-	go run . -d midi dv format/midi/testdata/midi/smpte-timecode.mid
-	go run . -d midi dv format/midi/testdata/midi/format-0.mid
+	go run . -d midi dv format/midi/testdata/midi/format-0x.mid
+	go run . -d midi dv format/midi/testdata/midi/format-0y.mid
+	go test ./format -run TestFormats/midi/testdata/format-0x.fqtest
+	go test ./format -run TestFormats/midi/testdata/format-0y.fqtest
```

Comment:
Don't see these files anywhere or maybe it's go test framework that mangles the test names somehow?

Reply:
Gah! It's in the _debug_ target of the _Makefile_ - they were the initial files I was testing with. Have fixed it to use the renamed versions (_format-0-extra-tracks_ and _format-0-extra-tracks2_).